src/store/plugins/fungibleTokens.js

```diff
@@ -65,7 +66,9 @@ export default (store) => {
           try {
             if (isEmpty(availableTokens)) return;
             const tokens = await fetchAllPages(
-              () => fetchFromMiddleware(`/v2/aex9/account-balances/${address}?limit=100`),
+              () => fetchFromMiddleware(urlWithParams(`/v2/aex9/account-balances/${address}`, {
```

Comment:
I don't think this change adds any benefit. Previous version was actually more readable.